content/synthetics/browser_check.md

```diff
@@ -0,0 +1,138 @@
+
+1. Optionally, select **Open in a pop-up** at the upper right of the page to open your test recording in a separate pop-up window in order to avoid sizing issues in the displayed window within Datadog's interface.
+2. Click on **Start recording** to begin recording your Browser Check.
+3. Your actions are recorded and used to create steps within your Browser Check Scenario.
+4. Use the actions available in the upper left corner to enrich your scenario:
+    {{< img src="synthetics/browser_check/browser_check_assertions.png" alt="Browser Check assertions" responsive="true" style="width:80%;">}}
+
+    **Note**: **Your last browser check step must be an Assertion**, otherwise there is nothing to check.
+5. Once you have finished your Scenario, click on **Save and Launch Test**.
+
+### Actions
+#### Assertion
+
+{{< img src="synthetics/browser_check/assertions_browser_check.png" alt="Assertion browser check" responsive="true" style="width:40%;">}}
+
+Assertions allow you to check if an element, a content, or some text is available in the current page:
+
+| Assertion                                               | Description                                                                                                                      |
+| ----                                                    | ----                                                                                                                             |
+| `Assert that an element is present on the page`         | Asserts that an element (such as a specific `span`, `div`, `h`, `a`, etc.) is present on the current page.                       |
+| `Check an element's content`                            | Makes sure that a specific element is located or not on the current page.                                                        |
+| `Assert that some text is present anywhere on the page` | Asserts that some specific text is present on the current page.                                                                  |
+| `Assert that some text is nowhere on the page`          | Asserts that some specific text is **NOT** present on the current page.                                                          |
+| `Check main page URL's content`                         | This takes the URL of the current page, and asserts whether a specific value (`string`, `number`, `regex`) is present within it. |
+
+#### Navigation
+
+The navigation action allow you to: 
+
+* Refresh the current page of the scenario.
+* Follow a specific link.
+
+#### Hover
+
+This browser check step isn’t added through an actual hovering mechanism (otherwise each element you are hovering would be added as a step) but using a dedicated action with a click.
+
+After selecting the Hover action, the hovering is blocked, and the step can be added by clicking on the element that you want to choose. 
```

Comment:
Not sure what `the hovering is blocked` means, maybe something like `After selecting the Hover action, click on the element you want to choose and a step will be created.`